src/libraries/Common/src/Interop/Windows/Kernel32/Interop.Globalization.cs

```diff
@@ -27,6 +27,7 @@ internal static unsafe partial class Kernel32
         internal const uint LOCALE_SNAME                = 0x0000005C;
         internal const uint LOCALE_INEUTRAL             = 0x00000071;
         internal const uint LOCALE_SSHORTTIME           = 0x00000079;
+        internal const uint LOCALE_ICONSTRUCTEDLOCALE   = 0x0000007d;
```

Comment:
Is this new flag supported on all the Windows versions that we support?

Reply:
The constructed locales is introduced on Windows Blue. This flag is supported since Windows Blue.

Reply:
Windows Blue — 8.1?  What happens on 7?  I believe 7 is out of support but I think we try to avoid had breaks there. IIRC we recently did some test passes there.

Reply:
This flag is used by the Windows API GetLocaleInfoEx(). The support of the flag is started on Windows Blue. If we call GetLocaleInfoEx with this flag on pre-Windows Blue (E.g. Windows 7), the call will fail and will return zero. And we'll work correctly 
In other word, there is no constructed locales pre-Windows blue.